terraform/transform_provider.go

```diff
@@ -340,22 +343,18 @@ func (n *graphNodeDisabledProviderFlat) ProviderName() string {
 
 // GraphNodeDependable impl.
 func (n *graphNodeDisabledProviderFlat) DependableName() []string {
-	return []string{n.Name()}
+	return modulePrefixList(
+		n.graphNodeDisabledProvider.DependableName(),
+		modulePrefixStr(n.PathValue))
 }
 
 func (n *graphNodeDisabledProviderFlat) DependentOn() []string {
 	var result []string
 
 	// If we're in a module, then depend on our parent's provider
 	if len(n.PathValue) > 1 {
-		prefix := modulePrefixStr(n.PathValue[:len(n.PathValue)-1])
```

Comment:
The build failure (terraform/transform_provider.go:357: undefined: prefix) seems related to this?